### ISSUE TITLE
Done in Sprint Defintion

### ISSUE THREAD

**Issue by dleeapho** [dleeapho/sandbox#11]
To capture metrics we might need to have a "Delivered in Sprint" swimlane so we can capture what was delivered in a sprint, get time-to-delivery metrics etc.

**Comment by ellismg** [comment]
The GitHub Issues API provides rich timing information about when issues were opened and closed, so it's not clear we need a "done in sprint" lane.  We could tool it after the fact.

**Comment by dleeapho** [comment]
@ellismg Agree, I should have been clear that the "Delivered in Sprint" was temporary in the sense that we could do it manually until we have solidified our process whereupon we could leverage the GitHub APIs to query the info directly.
